packages/manager/apps/dedicated/client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.controller.js

```diff
@@ -96,6 +96,9 @@ angular.module('App').controller(
         )
         .then((contracts) => {
           this.bindings.contracts.value = contracts;
+          if (!contracts.length) {
```

Comment:
```suggestion
          if (isArray(contracts) && !isEmpty(contracts)) {
```

Reply:
This was already approved https://github.com/ovh/manager/pull/3476